yew-macro/src/classes/mod.rs

```diff
@@ -0,0 +1,71 @@
+use proc_macro2::TokenStream;
+use quote::{quote, ToTokens};
+use syn::parse::{Parse, ParseStream};
+use syn::punctuated::Punctuated;
+use syn::{Expr, ExprLit, Lit, LitStr, Token};
+
+/// List of HTML classes.
+pub struct Classes(Punctuated<ClassExpr, Token![,]>);
+
+impl Parse for Classes {
+    fn parse(input: ParseStream) -> syn::Result<Self> {
+        input.parse_terminated(|x| ClassExpr::parse(x)).map(Self)
```

Comment:
This is fine.

Though I must say it's an optimization I find tedious and I often find clippy annoying for reporting something like that. Also because I find the closure more readable

Reply:
Fixed in 3482bf8